parts/kubernetesmastercustomscript.sh

```diff
@@ -215,7 +215,7 @@ function ensureJournal(){
 
 function ensureApiserver() {
     kubernetesStarted=1
-    for i in {1..600}; do
+    for i in {1..300}; do
```

Comment:
I would prefer to keep this at 10 minutes, for any slowness in container download or read/write latencies from disk.